pallets/pool-system/src/lib.rs

```diff
@@ -599,13 +624,28 @@ pub mod pallet {
 					Error::<T>::MinEpochTimeHasNotPassed
 				);
 
-				let (nav, nav_last_updated) = T::NAV::nav(pool_id).ok_or(Error::<T>::NoNAV)?;
+				// Get positive NAV from AUM
+				let (nav_aum, aum_last_updated) =
+					T::AssetsUnderManagementNAV::nav(pool_id).ok_or(Error::<T>::NoNAV)?;
+				ensure!(
+					now.saturating_sub(aum_last_updated) <= pool.parameters.max_nav_age,
+					Error::<T>::NAVTooOld
+				);
 
+				// Calculate fees to get negative NAV
+				T::OnEpochTransition::on_closing_mutate_reserve(
+					pool_id,
+					nav_aum,
+					&mut pool.reserve.total,
+				)?;
+				let (nav_fees, fees_last_updated) =
+					T::PoolFeesNAV::nav(pool_id).ok_or(Error::<T>::NoNAV)?;
 				ensure!(
-					now.saturating_sub(nav_last_updated) <= pool.parameters.max_nav_age,
+					now.saturating_sub(fees_last_updated) <= pool.parameters.max_nav_age,
```

Comment:
This can currently never happen, right? As we update the `FeesNAV` during `fn on_closing_mutate_reserve()`.

Reply:
Exactly! The `pallet-pool-fees` does not deal with any max age because `pallet-loans` doesn't do that either. I still wanted to make sure, both NAVs have the same max age boundaries until the impl of the accounting pallet.